CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changes by Version
 ==================
 
+0.14.0 (2020-11-09)
+-------------------
+* Bumped OpenTelemetry Collector to v0.13.0 ([#101](https://github.com/open-telemetry/opentelemetry-operator/pull/101), [@jpkrohling](https://github.com/jpkrohling)) 
```

Comment:
Why is this the same as the `v0.13.0` log?  should be `Bumped OpenTelemetry Collector to v0.14.0 ([#112](https://github.com/open-telemetry/opentelemetry-operator/pull/112)`  Is that right?

Reply:
I typically first open the PR and then amend with the right links, but forgot this last step this time.